covid_xprize/validation/scenario_generator.py

```diff
@@ -181,16 +187,38 @@ def do_main():
                         default="2021-06-19",
                         help="Last date of the scenario"
                              "Format YYYY-MM-DD. For example 2021-06-19")
+    parser.add_argument("-c", "--countries_path",
```

Comment:
The Phase 1 compliant daily task can be generated with the following additional arguments:
`-c` to specify the list of countries to take into consideration
`-p1` to make the list of countries compliant with Phase 1.

For instance:
```
-s 2020-12-22 -e 2021-06-19 -c countries_regions_phase1_fix.csv -p1 -o path/to/generated_scenario.csv
```